src/components/mainpage/chatscreen/components.mainpage.chatscreen.chatscreen.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import "./components.mainpage.chatscreen.chatscreen.css";
+import Header from "./components.mainpage.chatscreen.header";
+import ChatRoom from "./components.mainpage.chatscreen.chatroom";
+import Form from "./components.mainpage.chatscreen.form";
+
+function ChatScreen() {
+  const [user, setUser] = useState(0);
+  const [UserChat, changeChat] = useState([]);
+
+  const addChat = (user, chat) => {
+    const length = UserChat.length;
```

Comment:
`length`를 인덱스처럼 사용하고 계신데 변수 이름을 `index`로 변경하는 건 어떨까요?!
```suggestion
    const index = UserChat.length;
```